src/pages/RegistrationPage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import Form from '../components/Form';
+import EmailInput from '../components/EmailInput';
+import PasswordInput from '../components/PasswordInput';
+import NameInput from '../components/NameInput';
+import { Link } from 'react-router-dom';
+import { Colors } from '../assets/styles/colors';
+
+const RegistrationPage: React.FC = () => {
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+
+  const handleRegistration = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    console.log('First Name: ', firstName);
+    console.log('Last Name: ', lastName);
+    console.log('Email: ', email);
+    console.log('Password: ', password);
+
+    setFirstName('');
+    setLastName('');
+    setEmail('');
+    setPassword('');
+  };
+
+  return (
+    <Form
+      title="Registration"
+      onSubmit={handleRegistration}
+      submitButtonText="Register"
+      navigationText={
+        <>
+          Already have an account?{' '}
+          <Link to="/" style={{ textDecoration: 'none', color: Colors.green }}>
```

Comment:
I've noticed that both here and [here](https://github.com/SaltSquareIO/profile-app-react/pull/8/files#diff-5398e93fcf11b2ca16af33ab623e60bdcbf200391af9d6fe8f8cec9dc188b52cR30) you use the same style for the link. 
Try utilizing Mui's `createTheme()` fully to avoid writing duplicate code like this by creating a global style? Here's a [blog on it](https://muhimasri.com/blogs/mui-global-styles/)